src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -2127,5 +2127,94 @@
   "com.minecolonies.building.largequarry": "Large Quarry",
   "com.minecolonies.coremod.gui.quarry.desc": "To start the Quarry construct a Mine, hire a Quarrier and assign the Quarrier to the Quarry",
   "com.minecolonies.coremod.gui.workerhuts.druid": "Druid",
-  "item.minecolonies.mistletoe": "Mistletoe"
+  "item.minecolonies.mistletoe": "Mistletoe",
+
+  "minecolonies:builder.job.desc": "Constructs buildings and decorations.",
+  "minecolonies:deliveryman.job.desc": "Delivers requested items and blocks to players and colonists.",
+  "minecolonies:miner.job.desc": "Mines out a shaft system under the colony, gathering blocks and ores. Can be a dangerous Job",
+  "minecolonies:quarrier.job.desc": "Requires a Quarry building to mine a large hole. Safe job, but mostly returns common resources.",
+  "minecolonies:lumberjack.job.desc": "Cuts down trees and replants them based on the configured specifications.",
+  "minecolonies:farmer.job.desc": "Farms a large variety of crops at up 5 fields.",
+  "minecolonies:undertaker.job.desc": "Digs up citizens graves, gets their items with a chance to resurrect them.",
+  "minecolonies:fisherman.job.desc": "Spends most of the time fishing at a pond, river or lake.",
+  "minecolonies:baker.job.desc": "Handles all wheat based recipes. Knows some special bread recipes.",
+  "minecolonies:cook.job.desc": "Handles most of the cooking and hands out food to colonists.",
+  "minecolonies:shepherd.job.desc": "Shears sheep and controls the size of the flock.",
+  "minecolonies:cowboy.job.desc": "Gets milk and meat.",
+  "minecolonies:swineherder.job.desc": "Provides the colony with pork.",
+  "minecolonies:chickenherder.job.desc": "Gathers eggs and feathers and other chicken products.",
+  "minecolonies:ranger.job.desc": "Distance fighter, high damage, low survival chance in close combat.",
+  "minecolonies:knight.job.desc": "Close quarters fighter. High resilience, low damage output.",
+  "minecolonies:composter.job.desc": "Composts a lot of the colony waste products",
+  "minecolonies:student.job.desc": "Reads books to increase their intelligence. High intelligence allows learning other skills faster.",
+  "minecolonies:archertraining.job.desc": "Archer Trainee. Practices distance fighting.",
+  "minecolonies:combattraining.job.desc": "Knight Trainee. Practices close quarter fighting.",
+  "minecolonies:sawmill.job.desc": "Provides the colony with most wood based items.",
+  "minecolonies:blacksmith.job.desc": "Provides the colony with most tools, armor and weapons.",
+  "minecolonies:stonemason.job.desc": "Provides most stone variants.",
+  "minecolonies:stonesmeltery.job.desc": "Provides the stone variants that require smelting and charcoal",
+  "minecolonies:crusher.job.desc": "Crushes cobble to gravel, gravel to sand and sand to clay.",
```

Comment:
"Crushes cobble to gravel, gravel to sand, and sand to clay."
+1 for the Oxford comma!